snuba/snuba_migrations/uptime_monitor_checks/0001_uptime_monitor_checks.py

```diff
@@ -0,0 +1,95 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import AddIndicesData, OperationTarget, SqlOperation
+
+storage_set = StorageSetKey.UPTIME_MONITOR_CHECKS
+table_prefix = "uptime_monitor_checks"
+local_table_name = f"{table_prefix}_local"
+dist_table_name = f"{table_prefix}_dist"
+
+## what about all the fancy codecs? do we need those?
+columns: List[Column[Modifiers]] = [
+    Column("organization_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("environment", String(Modifiers(nullable=True, low_cardinality=True))),
+    Column("uptime_subscription_id", UInt(64)),
+    Column("uptime_check_id", UUID()),
+    Column("scheduled_check_time", DateTime()),
+    Column("timestamp", DateTime()),
+    Column("_sort_timestamp", DateTime()),
+    Column("duration", UInt(64)),
+    Column("region_id", UInt(16, Modifiers(nullable=True))),
+    Column("check_status", String(Modifiers(low_cardinality=True))),
+    Column(
+        "check_status_reason",
+        String(Modifiers(nullable=True, low_cardinality=True)),
+    ),
+    Column("http_status_code", UInt(16)),
+    Column("trace_id", UUID()),
+    Column("retention_days", UInt(16)),
+]
+
+indices: Sequence[AddIndicesData] = [
```

Comment:
I wouldn't start with adding indices already. If you don't want to put the `trace_id` in the sort key, fine, but this might not be necessary.

Reply:
i've added the trace_id to the sort key / primary key